### PR TITLE
unit test changes and adb sync command addition

### PR DESCRIPTION
**Pull request by vvscloud**

### CODE REVIEW

**Comment by vvscloud** [comment]
I have removed kill-server and start-server testing from unit test code. Generally kill-server followed by start-server will put our code into infinite wait and its behavior is unpredictable. Please merge this code.

File: adb_android/adb_android.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def wait_for_device():
 
 def sync():
     """
-    Perform a sync with device and host after a series of commands executed
+    Perform a sync with device and host
```
Comment (vmalyi) [inline adb_android/adb_android.py]: Let's be more precise here by specifying what exactly will be synchronized. Android docs say 

> "copy host->device only if changed"

File: tests/test_adb_android.py [hunk 2]
```diff
@@ -252,24 +252,11 @@ def test_wait_for_device(self):
...
-        result = adb.start_server()
-        result[0].should.be(POSITIVE_EXP_RESULT)
-
-
 class TestGetState(unittest.TestCase):
     @unittest.skipIf(is_emulator(), 'skip if run on emulator')
     def test_get_state(self):
         result = adb.get_state()
-        result[1].should.match(r'device\n')
+        result[1].should.match(r'device')
```
Comment (vmalyi) [inline tests/test_adb_android.py]: What about adding a test class for "sync" command also?

File: adb_android/adb_android.py (hunk 1 above)
Comment (vvscloud) [inline adb_android/adb_android.py]: yes, makes sense; will update the doc-string accordingly

File: tests/test_adb_android.py (hunk 2 above)
Comment (vvscloud) [inline tests/test_adb_android.py]: will add the testcase for sync

**Comment by vmalyi** [comment]
Thanks!

**Pull request closed without merging**